src/library_wasmfs.js

```diff
@@ -64,7 +65,15 @@ mergeInto(LibraryManager.library, {
     },
     createPath: (parent, path, canRead, canWrite) => {
       // Cache file path directory names.
-      wasmFSPreloadedDirs.push({parentPath: parent, childName: path});
+      var parts = path.split('/').reverse()
+      while (parts.length) {
+        var part = parts.pop();
+        if (!part) continue;
+        var current = PATH.join2(parent, part);
```

Comment:
(Sad to need `PATH` just for this... but I don't understand the context here yet.)

Reply:
Indeed, but this does seems like the quickest (best?) way to get feature parity with the current/legacy `FS.createPath`.

This code is taken almost verbatim from `FS.createPath`.

Reply:
Because in my new code in `_emscripten_fs_load_embedded_files` I'm relying on the existing behaviour of `FS.createPath`, which is that it can recursively create directories.

The current wasmfs version did not support this, and was arguably broken in this respect.  So

Reply:
Agreed, this seems best for now. We'll need to rethink it eventually when we focus on code size for wasmfs, but that's not urgent.